.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
Ran into an issue where actool fails to load libpng16 on CentOS 7, which has libpng15 as the default. I think having libpng as a submodule would be worth revisiting.